src/add-to-app/android/project-setup.md

```diff
@@ -134,45 +155,69 @@ module an embeddable Android library.
 {{site.alert.end}}
 
 {{site.alert.note}}
-  To avoid Dex merging issues, `flutter.androidPackage` should not be identical to your host app's package name
+  To avoid Dex merging issues, `flutter.androidPackage` should
+  not be identical to your host app's package name.
 {{site.alert.end}}
 
-### Java 11 requirement
+#### Java version requirement
+{:.no_toc}
 
-The Flutter Android engine uses Java 11 features.
+The Flutter Android Gradle plugin uses features found in Java 11 or later.
```

Comment:
While it is technically related to Gradle in this context, I believe it is more correct to say the "Flutter Android engine" here since it uses the features as well. Could we be less specific and just say something like:

```suggestion
Flutter requires your project to declare compatibility with Java 11 or later.
```